worldengine/imex/__init__.py

```diff
@@ -12,24 +12,49 @@
 import sys
 
 
-def export(world, export_type, export_bpp, export_signed, export_normalize):
+def export(world, export_filetype, export_bpp, export_signed, path = 'seed_output', export_raw = False):
```

Comment:
I did not rename your variable, I removed "export_normalize". When requesting integer output without scaling one would end up with numbers between 0 and maybe 15 - that kind of output is useless, especially when blown up to a 32 Bit image, so I enabled normalization by default. However, when requesting "raw" output (float32), normalization is detrimental at best, as you said.

Full GDAL is difficult in my opinion, I had a hard time installing it. PyPNG, however, is a nice middle ground. At least it is more capable than Pillow, although it is restricted to PNG. (Which I think is fine since that is a widely-spread, lossless format.)